src/gen_multisig/gen_multisig.cpp

```diff
@@ -74,8 +74,6 @@ namespace
   const command_line::arg_descriptor<bool, false> arg_testnet = {"testnet", genms::tr("Create testnet multisig wallets"), false};
   const command_line::arg_descriptor<bool, false> arg_stagenet = {"stagenet", genms::tr("Create stagenet multisig wallets"), false};
   const command_line::arg_descriptor<bool, false> arg_create_address_file = {"create-address-file", genms::tr("Create an address file for new wallets"), false};
-
-  const command_line::arg_descriptor< std::vector<std::string> > arg_command = {"command", ""};
```

Comment:
Not sure why you'd want to remove the command argument here, seems like a mistake.

Reply:
As far as I can tell that argument was not in use at all. If you look below, it is never added as an argument like the others. 

https://github.com/monero-project/monero/blob/56bbfd2921cc9f2a3a619de1481c01f28de01bc3/src/gen_multisig/gen_multisig.cpp#L166-L173

Reply:
My bad, misread the surrounding context.